packages/rdr/rdr.2.0/descr

```diff
@@ -0,0 +1,17 @@
+Rdr is a cross-platform binary analysis and reverse engineering library, utilizing a unique symbol map for global analysis.
+
+`rdr` is an OCaml tool/library for doing cross-platform analysis of binaries, by printing headers, locating entry points, showing import and export symbols, their binary offsets and size, etc.
```

Comment:
It would be nice to have the text wrapped to 78 chars (opam does not wrap it when using `opam show`).